.gitignore

```diff
@@ -148,3 +148,4 @@ axe_reports
 
 # Build output
 html
+.vscode/launch.json
```

Comment:
Could this be moved up to line 136 and perhaps we could just have .vscode to exclude all vscode files?